docs/source/types-testing.rst

```diff
@@ -234,3 +219,93 @@ Or we can simplify further by using ``assertHasMessage`` and ``assertHasAction``
             self.assertEqual(action.children, messages)
             # Each message had the respective server set.
             self.assertEqual(servers, [msg.message["server"] for msg in messages])
+
+
+Restriction Testing to Specific Messages
+----------------------------------------
+
+If you want to only look at certain messages when testing you can log to a specific ``eliot.Logger`` object rather than the global one.
+
+You can log messages to a specific ``Logger``:
+
+.. code-block:: python
+
+    from eliot import Message, Logger
+
+    class YourClass(object):
+        logger = Logger()
```

Comment:
(My guess is almost no one will ever create a `Logger` ever again once this is merged.)